### ISSUE TITLE
Suggestion: Deprecate old poll type

### ISSUE THREAD

**Issue by IstoraMandiri** [twitter-together/action#215]
Now that FrontMatter is supported and includes polls, to simplify documentation and standardize usage, it may be worth deprecating (and/or dropping from the documentation) the ability to use the original poll format. I.e.

**Do not recommend this version**

```tweet
What is your favorite color?

( ) Red
( ) Blue
( ) Green
```

**Instead, only show / support this version**

```tweet
---
poll:
  - Red
  - Blue
  - Green
---

What is your favorite color?
```

**Comment by MattIPv4** [comment]
I disagree with removing the old body syntax for polls -- in fact, we even have an issue open to look at adding body syntax for media as well, which was pulled out of the original frontmatter thread: https://github.com/twitter-together/action/issues/200

**Comment by gr2m** [comment]
I'm leaning towards deprecating the current poll style, too, for the sake of lowering complexity and maintenance. I think it's simple enough to document how to add polls with front-matter. The original idea for `^` came before we discussed front-matter, I could live without that as well. I think it will become more clear that way. And as we define a syntax to define a tweet, keeping it simple and well tested will make it more likely for other projects to pick it up as well, which would be pretty great.
